contracts/System.sol

```diff
@@ -3,38 +3,41 @@ pragma solidity 0.8.4;
 
 import "./interface/IRelayerHub.sol";
 
-contract System {
+abstract contract System {
 
   bool public alreadyInit;
+  bool public s_guardIsActive;
```

Comment:
as discussed in prior pr this type of slot-packing is not recommended but will not result in a collision